src/analysis/souffle/authorization_logic.dl

```diff
@@ -52,12 +52,33 @@ isPrincipal(speaker) :- saysDowngrades(speaker, _, _).
 //-----------------------------------------------------------------------------
 downgrades(path, tag) :- ownsTag(owner, tag), saysDowngrades(owner, path, tag).
 
+// This rule conservatively says that if _any_ principal claims that a path has
+// a tag, then it has the tag according to the taint analysis. This rule is
+// likely to be adjusted later. For confidentiality tags this should be OK to
+// start with.
+claimHasTag(path, tag) :- ownsTag(owner, tag), saysHasTag(_, path, tag).
```

Comment:
This is in conflict with #121 , which introduces a `claimHasTag` with a `Principal`.

Reply:
Ah, I must have missed this piece when resolving the rebase. I made it just `claimHasTag(p, q, t) :- saysHasTag(p, q, t)` so that we can still pull facts from the auth logic implementation (since it emits things beginning with says and these are still somewhat separate pieces of software for this MVP).